splash-dao-offchain/src/constants.rs

```diff
@@ -75,6 +75,8 @@ pub mod fee_deltas {
     pub const REDEEM_VOTING_ESCROW_FEE_DELTA: u64 = 320_000;
 }
 
+pub const VOTING_ESCROW_TX_TTL: u64 = 300;
```

Comment:
Forgot to incorporate this from your earlier review (https://github.com/splashprotocol/splash-offchain-multiplatform/pull/176#discussion_r1980961435)